docs/config.toml

```diff
@@ -5,6 +5,10 @@ description = "Everything you need to make a static site engine in one binary."
 compile_sass = true
 build_search_index = true
 
+taxonomies = [
+  { name = "tags"}
```

Comment:
they should probably be named theme_tag no? In case we ever want other types of tags